pkg/kubelet/util/format/pod.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package util
+package format
```

Comment:
(optional) I think it might be good to explicitly call out that this is a logging utility by including that in the name, such as `log` (we don't use the standard library log anywhere in kubelet, so that name won't collide).